test/Makefile.am

```diff
@@ -83,19 +84,33 @@ check_PROGRAMS                                += \
     $(NULL)
 endif # NLER_BUILD_FLOW_TRACER
 
-if NLER_BUILD_UTILITIES
-check_PROGRAMS                                += \
-    test-settings                                \
-    $(NULL)
-endif # NLER_BUILD_UTILITIES
-
 if !NLER_BUILD_EVENT_TIMER
 check_PROGRAMS                                += \
     test-subpub                                  \
     test-timer                                   \
     $(NULL)
 endif # !NLER_BUILD_EVENT_TIMER
 
+# Test applications that should be built but not installed and not run
+# when the 'check' target is run but should always be built to ensure
+# overall "build sanity".
```

Comment:
Maybe a cut and paste error here?  Looks like this should be something like:

    Applications that should not be installed or run when the 'check' target is run,
    but which should always be built to ensure overall "build sanity".